src/Validation/Validator.php

```diff
@@ -40,6 +45,10 @@ private function checkConstraint(
 
     public function validate(Token $token, Constraint ...$constraints): bool
     {
+        if (count($constraints) === 0) {
```

Comment:
I know this is kind of micro optimisation, but let's avoid using `count()` here.
We can do `if ($constraints === [])`.